src/extensions/default/BrambleUrlCodeHints/camera/selfieWidget.html

```diff
@@ -8,8 +8,29 @@
     </div>
     <div id="selfie-gallery">
         <div class="polaroid">
+        <!--<div>-->
             <img id="selfie-photo" />
         </div>
+
+    </div>
+    <div>
+      <canvas id="canvas"></canvas>
+    </div>
+    <div id="filter-list">
+        <nav>
+          <button id="resetbtn" class="btn btn-success">Reset Photo</button>
+          <button id="pinholebtn" value="pinhole" class="btn btn-primary">Pin Hole</button>
+          <button id="sepiabtn" value="sepia" class="btn btn-primary">Sepia</button>
+          <button id="contrastbtn" value="contrast" class="btn btn-primary">Contrast</button>
+        </nav>
+        <br/>
+        <nav>
+          <button id="vintagebtn" value="vintage" class="btn btn-primary">Vintage</button>
+          <button id="sunrisebtn" value="sunrise" class="btn btn-primary">Sun Rise</button>
+          <button id="glowingsunbtn" value="glowingSun" class="btn btn-primary">Glow Sun</button>
+          <button id="embossbtn" value="emboss" class="btn btn-primary">Emboss</button>
+        </nav>
     </div>
 </div>
+<script type="text/javascript" src="/node_modules/caman/dist/caman.full.min.js"></script>
```

Comment:
We're going to want to move this file directly into the extension's dir vs. requireing from `node_modules`.  A lot of extensions include a `thirdparty/` dir, and you can put it there.  Make sure the license info and copyright are also included, if not in the `min.js` version, then include the project's license file in `thirdparty/caman/*`.